### ISSUE TITLE
Linter doesn't seem to catch hardcoded strings if using backticks

### ISSUE THREAD

**Issue by gary-cohen** [edvardchen/eslint-plugin-i18next#85]
We've noticed if a hardcoded string is used, but with backticks, the linter doesn't catch it as being a hardcoded string.

e.g.

✅ **Caught**
```<Text>{booleanTest ? "Foo" : "Bar"}</Text>```

❌  **Uncaught**
```<Text>{booleanTest ? `Foo ${bar}` : `Baz ${zip}`}</Text>```

Thanks.

**Comment by edvardchen** [comment]
The document here seemed not clear enough. You can enable the option `should-validate-template`

**Comment by mathieutu** [comment]
Hey. I missed it too.

Why not enable it by default like other strings? If because of potential params, couldn't we detect via the ast if they are hard-coded?

**Comment by reckter** [comment]
This broke our linting while using http://relay.dev.
(and I imagine other graphql libraries as well, that rely on tagged-templating).

It showed `disallow literal string`  errors for every `graphql`-tagged template, event if not in a `.tsx` file at all.

I think this should either not detect these tagged-templates outside of jsx statements at all, or at least have an option to allow certain tagged-templates.

**Comment by edvardchen** [comment]
Well I cannot satisfy everyone and I will stick to my original thought and roll back to the original behavior. And I don't want to have two opinions like 'should-validate-template-with-expression' and 'should-validate-template-without-expression' rather than the existing one. 

The backtick can serve a cheap way to bypass the checking if the user wants to instead of carrying on the heavy configuration. The linting is to help people succeed and not to cause more problems.

**Comment by edvardchen** [comment]
@reckter try `v6.0.0-8` and it won't warn the graphql query string any more except that you enable the `should-validate-template` option
